v2/spanner-to-sourcedb/README.md

```diff
@@ -0,0 +1,363 @@
+
+# Cloud Spanner Reverse Replication User Guide
+
+## Overview
+
+### Background
+
+Migrating a database is a complex affair, involving changes to the schema, converting the application, tuning for performance, ensuring minimal downtime and completeness during data migration. It is possible that after migration and cutover, issues/inconsistent performance are encountered on the target (Cloud Spanner) requiring a fallback to the original source database with minimal disruption to the service. Reverse replication enables this fallback by replicating data written on Cloud Spanner back to the source database.This allows the application to point to the source and continue serving requests correctly.
+
+Reverse replication could also be used to replicate the Cloud Spanner writes to a different database, other than the source database, for performing reconciliation, validations and reporting.
+
+### How it works
+
+Reverse replication flow involves below steps:
+
+1. Reading the changes that happened on Cloud Spanner using [Cloud Spanner change streams](https://cloud.google.com/spanner/docs/change-streams)
+2. Removing forward migrated changes ( if configured to filter )
+3. Transforming Cloud Spanner data to source database schema
+4. Verifying that the source database does not already have more up-to-date data for the given primary key
```

Comment:
Rephrase to : Verifying whether the source database already contains more recent data for the specified primary key.

Reply:
Done